.ado/azure-pipelines.yml

```diff
@@ -39,7 +39,7 @@ jobs:
   - job: AndroidPR
     displayName: Android PR
     pool:
-      vmImage: 'macos-11'
+      vmImage: 'macos-latest'
```

Comment:
nit, could use `$(VmImageApple) which is defined as `macos-latest-internal` here too

Reply:
won'tfixing but will mind for further reflection -- if our previous problem was pinning the version not sure an internal pinning (?) is really doing us a favor